sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -680,4 +687,21 @@ class ExpressionParserSuite extends PlanTest {
     assertEqual("last(a ignore nulls)", Last('a, Literal(true)).toAggregateExpression())
     assertEqual("last(a)", Last('a, Literal(false)).toAggregateExpression())
   }
+
+  test("timestamp literals") {
+    withSQLConf(SQLConf.SESSION_LOCAL_TIMEZONE.key -> "UTC") {
```

Comment:
To check if a selected timezone reflected in a parser, how about adding tests for other timezones?